pkg/kubelet/kubelet.go

```diff
@@ -1489,6 +1498,15 @@ func (kl *Kubelet) killUnwantedPods(desiredPods map[types.UID]empty,
 	return running, utilErrors.NewAggregate(errs)
 }
 
+func findPod(pods []*api.Pod, name string) *api.Pod {
+	for ix := range pods {
```

Comment:
What is the effect of a prestop hook failing?

On Tue, Apr 28, 2015 at 1:27 PM, Brendan Burns <notifications@github.com>
wrote:

> In pkg/kubelet/kubelet.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7421#discussion_r29266352>
> :
>
> > @@ -1489,6 +1498,15 @@ func (kl *Kubelet) killUnwantedPods(desiredPods map[types.UID]empty,
> >  	return running, utilErrors.NewAggregate(errs)
> >  }
> >
> > +func findPod(pods []*api.Pod, name string) *api.Pod {
> > +	for ix := range pods {
>
> added a TODO, though I'm really not sure there is measurable win here,
> given expected # of pods / machine.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7421/files#r29266352>
> .
>